src/Cli/dotnet/commands/dotnet-workload/uninstall/WorkloadUninstallCommand.cs

```diff
@@ -43,7 +43,7 @@ internal class WorkloadUninstallCommand : WorkloadCommandBase
             userProfileDir = userProfileDir ?? CliFolderPathCalculator.DotnetUserProfileFolderPath;
             _sdkVersion = WorkloadOptionsExtensions.GetValidatedSdkVersion(parseResult.GetValueForOption(WorkloadUninstallCommandParser.VersionOption), version, dotnetPath, userProfileDir, true);
 
-            var workloadManifestProvider = new SdkDirectoryWorkloadManifestProvider(dotnetPath, _sdkVersion.ToString(), userProfileDir);
+            var workloadManifestProvider = new SdkDirectoryWorkloadManifestProvider(dotnetPath, _sdkVersion.ToString(), userProfileDir, SdkDirectoryWorkloadManifestProvider.GetGlobalJsonPath(Environment.CurrentDirectory));
```

Comment:
Can this be moved into a base constructor? Feels like a lot of repetition.

Reply:
Taking a look...

Reply:
Looks consistent to me. All the default behavior appears to be on the factory side.